src/django/api/models/facility/facility.py

```diff
@@ -354,19 +354,3 @@ def current_tile_cache_key():
             tile_version = 0
 
         return f'{timestamp}-{tile_version}'
-
-    def activity_reports(self):
```

Comment:
Could you explain why did you remove ` activity_reports` function?

Reply:
+1

Reply:
Since this function is related to activity reports I performed a small refactoring and simply removed an unused function.

Reply:
Ack